src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -2620,6 +2557,84 @@ internal void EnsureAnonymousTypeTemplates(CancellationToken cancellationToken)
             }
 
             return true;
+
+
+        }
+
+        Stream GetRefPeStream(DiagnosticBag metadataDiagnostics, EmitStreamProvider metadataPEStreamProvider) 
+        {
+            if (metadataDiagnostics.HasAnyErrors())
+            {
+                return null;
+            }
+
+            var refPeStream = metadataPEStreamProvider.GetOrCreateStream(metadataDiagnostics);
+            Debug.Assert(refPeStream != null || metadataDiagnostics.HasAnyErrors());
+            return refPeStream;
+        }
+
+        Stream GetPortablePdbStream(DiagnosticBag metadataDiagnostics, EmitStreamProvider pdbStreamProvider)
```

Comment:
`private static`

Reply:
`GetRefPeStream` and `GetPortablePdbStream` look identical.

Reply:
I've collapsed both of these to GetAuxStream